compiler/p/codegen/OMRMachine.cpp

```diff
@@ -576,8 +576,7 @@ TR::RealRegister *OMR::Power::Machine::freeBestRegister(TR::Instruction     *cur
       {
       // XXX: Reusing backing stores here is causing the virtual being spilled to be clobbered
       case TR_GPR:
-         if (false /*!cg()->comp()->getOption(TR_DisableOOL) && (cg()->isOutOfLineColdPath() || cg()->isOutOfLineHotPath()) &&
-                virtReg->getBackingStorage()*/)
+         if (false /*(cg()->isOutOfLineColdPath() || cg()->isOutOfLineHotPath()) && virtReg->getBackingStorage()*/)
```

Comment:
Not sure how I feel about these being disabled, but we'll get to them. We seem to have the same code on other backends but this `if` is enabled there. I didn't change anything here semantically, just thought I'd point this out to people looking at this PR.

Reply:
Heh about to ask why we can't just remove this code if it's disabled. But I'm ok with it being cleaned up in another PR.